telemetry_datarates.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python

```

Comment:
.... and should probably be in the tools directory...

It should also possibly be renamed to "pytelemetry_rates.py" or something so it can be installed :-)